interface-definitions/vpn_openconnect.xml.in

```diff
@@ -13,25 +13,43 @@
               <help>Authentication for remote access SSL VPN Server</help>
             </properties>
             <children>
-              <leafNode name="mode">
+              <node name="mode">
                 <properties>
                   <help>Authentication mode used by this server</help>
-                  <valueHelp>
-                    <format>local</format>
-                    <description>Use local username/password configuration</description>
-                  </valueHelp>
-                  <valueHelp>
-                    <format>radius</format>
-                    <description>Use RADIUS server for user autentication</description>
-                  </valueHelp>
-                  <constraint>
-                    <regex>^(local|radius)$</regex>
-                  </constraint>
-                  <completionHelp>
-                    <list>local radius</list>
-                  </completionHelp>
                 </properties>
-              </leafNode>
+                <children>
+                  <leafNode name="local">
+                    <properties>
+                      <help>Use local username/password configuration (OTP supported)</help>
+                      <valueHelp>
+                        <format>password</format>
+                        <description>Password-only local authentication (default)</description>
+                      </valueHelp>
+                      <valueHelp>
+                        <format>otp</format>
+                        <description>OTP-only local authentication</description>
+                      </valueHelp>
+                      <valueHelp>
+                        <format>password-otp</format>
+                        <description>Password (first) + OTP local authentication</description>
+                      </valueHelp>
+                      <constraint>
+                        <regex>^(password|otp|password-otp)$</regex>
+                      </constraint>
+                      <constraintErrorMessage>Invalid authentication mode</constraintErrorMessage>
```

Comment:
Maybe include a list of valid modes in the message? "Must be one of: ...".

Reply:
Fixed